src/webgpu/shader/validation/parse/break_if.spec.ts

```diff
@@ -0,0 +1,133 @@
+export const description = `Validation tests for break if`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { keysOf } from '../../../../common/util/data_tables.js';
+import { ShaderValidationTest } from '../shader_validation_test.js';
+
+export const g = makeTestGroup(ShaderValidationTest);
+
+const kTests = {
+  loop_break: {
+    src: 'loop { break if true; }',
+    pass: false,
+  },
+  loop_if_break: {
+    src: 'loop { if true { break if false; } }',
+    pass: false,
+  },
+  continuing_break_if: {
+    src: 'loop { continuing { break if (true); } }',
```

Comment:
Drop the parentheses?

Reply:
You can drop the `()`s, but it's also valid for `break if` to use them. So, I think we should have tests which have both styles.

I added a second test which does the same thing without parens.

Reply:
LGTM, thanks